### PR TITLE
Issue#47

### PR DESCRIPTION
**Pull request by Parabellum1905y**

### CODE REVIEW

File: scorex/src/main/scala/scorex/core/network/PeerConnectionHandler.scala [hunk 1]
```diff
@@ -172,6 +172,7 @@ class PeerConnectionHandler(val settings: NetworkSettings,
         messagesHandler.parseBytes(packet.toByteBuffer, Some(selfPeer.get)) match {   //todo: .get
           case Success(message) =>
             log.info("Received message " + message.spec + " from " + remote)
+            log.debug(s"${networkControllerRef.path.name}")
```
Comment (Fly-Style) [inline scorex/src/main/scala/scorex/core/network/PeerConnectionHandler.scala]: Could you please generalize this debug message? 
Smth like that :
`log.debug(s"NetworkController actor name : ${networkControllerRef.path.name}")` ?

File: src/test/scala/network/BlockchainDownloaderTest.scala [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+import java.io.File
+
+import com.dimafeng.testcontainers.{DockerComposeContainer, ForAllTestContainer}
+import org.scalatest.FunSuite
+
+/**
+  * @author luger. Created on 09.04.18.
+  * @version ${VERSION}
+  */
+class BlockchainDownloaderTest extends FunSuite  with ForAllTestContainer{
```
Comment (Fly-Style) [inline src/test/scala/network/BlockchainDownloaderTest.scala]: Will you finish it?

File: scorex/src/main/scala/scorex/core/network/PeerConnectionHandler.scala (hunk 1 above)
Comment (Parabellum1905y) [reply scorex/src/main/scala/scorex/core/network/PeerConnectionHandler.scala]: fixed, removed debug output

File: src/test/scala/network/BlockchainDownloaderTest.scala (hunk 2 above)
Comment (Parabellum1905y) [reply src/test/scala/network/BlockchainDownloaderTest.scala]: nope, temporary removed.

**Pull request merged**